test/test_effect_classes.py

```diff
@@ -75,7 +88,9 @@ def test_start_loss():
     expect_effect(
         variant,
         transcript_id="ENST00000420316",
-        effect_class=StartLoss)
+        effect_class=StartLoss,
+        modifies_coding_sequence=True,
+        modifies_protein_sequence=True)
```

Comment:
`StartLoss` modifies_protein_sequence? I guess it does, but tbh not what I would have expected when filtering on `modifies_protein_sequence` so good to know!